superset/typing.py

```diff
@@ -25,7 +25,7 @@
     str, str, Optional[str], Optional[str], Optional[int], Optional[int], bool
 ]
 DbapiDescription = Union[List[DbapiDescriptionRow], Tuple[DbapiDescriptionRow, ...]]
-DbapiResult = List[Union[List[Any], Tuple[Any, ...]]]
+DbapiResult = Sequence[Union[List[Any], Tuple[Any, ...]]]
```

Comment:
for some reason `MyPy` was giving me errors on this locally (I'm probably running a newer version than CI). As I know some Dbapis are actually returning tuples, I think this makes sense anyway.